lib/importer/sheets.js

```diff
@@ -0,0 +1,10 @@
+const xlsx = require("node-xlsx").default;
+
+exports.ListSheets = (filename) => {
+  const worksheet = xlsx.parse(filename);
+  const sheets = worksheet.map((sheet) => {
+    return sheet.name;
+  });
+
+  return sheets;
+};
```

Comment:
It might make sense to parse this upfront upon upload at some point, and storing the metadata required, rather than parsing it on every call.

Reply:
I agree.  I've only done it like this to make progress on the template, am expecting @alaric-rd  may want to change the 'api'.